### PR TITLE
cmd, core: remove calaveras testnet 

### PR DESCRIPTION
**Pull request by Zachinquarantine**
The Calaveras testnet has been shut down, because it was only a short lived testnet in the first place. This PR fully removes Calaveras.

### CODE REVIEW

**Comment by karalabe** [comment]
The code doesn't compile.

**Comment by Zachinquarantine** [comment]
The remaining build problems seem to be caused by [golangci/golangci-lint/1462](https://github.com/golangci/golangci-lint/issues/1462)

**Comment by Zachinquarantine** [comment]
ohh.... I see the error now lmao
slika
goerli is missing a space

**Comment by Zachinquarantine** [comment]
I can't figure out how this is failing...

**Comment by Zachinquarantine** [comment]
> 
> 
> I can't figure out how this is failing...

I literally copied the failing code from ``master``, which passed, and yet somehow on this branch it fails.

File: core/state/snapshot/snapshot.go [hunk 1]
```diff
@@ -168,18 +168,11 @@ type Tree struct {
...
-//
-// - if the 'recovery' parameter is true, all memory diff-layers will be discarded.
-//   This case happens when the snapshot is 'ahead' of the state trie.
-// - otherwise, the entire snapshot is considered invalid and will be recreated on
-//   a background thread.
+// If the snapshot is missing or the disk layer is broken, the entire is deleted
+// and will be reconstructed from scratch based on the tries in the key-value
+// store, on a background thread. If the memory layers from the journal is not
+// continuous with disk layer or the journal is missing, all diffs will be discarded
+// iff it's in "recovery" mode, otherwise rebuild is mandatory.
```
Comment (holiman) [inline core/state/snapshot/snapshot.go:175]: this change doesn't belong here

File: params/config.go [hunk 2]
```diff
@@ -27,11 +27,10 @@ import (
 
 // Genesis hashes to enforce below configs on.
 var (
-	MainnetGenesisHash   = common.HexToHash("0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3")
+        MainnetGenesisHash   = common.HexToHash("0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3")
```
Comment (holiman) [inline params/config.go:30]: Some whitespace changed here, intentional?

Comment (Zachinquarantine) [reply params/config.go:30]: I copied those strings from ``master``, that's how they were there...

File: core/state/snapshot/snapshot.go (hunk 1 above)
Comment (Zachinquarantine) [reply core/state/snapshot/snapshot.go:176]: fixed!

**Comment by holiman** [comment]
> I literally copied the failing code from master, which passed, and yet somehow on this branch it fails.

Yeah, things become a bit screwy when one branches off a certain commit, and then merge the newer commits into your branch. We usually instead rebase the branch to be on top of master. 

I've rebased this on top of master now, and squashed your commits, and added one commit on top to fix a remaining lint issue. If you want to push more commits, you'll need to `git pull` the branch, and in order to actually "accept" the rebased remote branch, you'll need to `git reset --hard origin/Calaverasgobrr`

**Pull request merged**